src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -125,6 +127,7 @@ static string LocalAddress(IBuilder builder)
             return builder.Build<ReadOnlySettings>().LocalAddress();
         }
 
+        /// Not sure about this one
```

Comment:
Uses `IBuilder` to acquire the `ISubscriptionStorage`. I'm sure there is a better design. Thoughts @janovesk @SzymonPobiega ?